README.md

```diff
@@ -1,6 +1,12 @@
 # BridgeEx
 
-A library to build bridges to other services (actually only graphql ones are supported).
+[![Module Version](https://img.shields.io/hexpm/v/bridge_ex.svg)](https://hex.pm/packages/bridge_ex)
```

Comment:
Maybe could be useful to add drone badge?
[![Build Status](https://drone-1.prima.it/api/badges/primait/bridge_ex/status.svg)](https://drone-1.prima.it/primait/bridge_ex)

Reply:
Yes! You are right